draft-ietf-tls-tls13.md

```diff
@@ -2322,6 +2340,13 @@ secp256r1, etc.
   X9.62 {{X962}} and FIPS 186-4 {{DSS}}.
   Values 0xFE00 through 0xFEFF are reserved for private use.
 
+x25519, etc.
+: Indicates support of the corresponding ECDH function.
+
+Ed25519, etc.
+: Indicates support of the corresponding curve for
+  signatures.
+
```

Comment:
I would name these differently (this may be a comment for the otehr document)

ecdhe_x25519

and 
eddsa_25519